manager/rpc/license.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package license;
+
+message PollingRequest{
+	string vendor = 1; // Xilinx or Cadence
```

Comment:
If you already know these values, they should be enums. Means you don't have spend time string validating.